cms/search/views.py

```diff
@@ -43,6 +43,15 @@ def get_queryset(self) -> list[Page]:
         excluding pages we do not want to index.
         """
         excluded_classes = [get_model_by_name(name) for name in settings.SEARCH_INDEX_EXCLUDED_PAGE_TYPES]
-        qs: list[Page] = Page.objects.live().public().not_exact_type(*excluded_classes).specific().defer_streamfields()
+        included_locales = [code.strip().lower() for code in settings.SEARCH_INDEX_INCLUDED_LANGUAGES if code.strip()]
```

Comment:
Nite, you do this in 2 places, use the same func?. Move to utils?.

Reply:
Was thinking of this.

Superseded: https://github.com/ONSdigital/dis-wagtail/pull/441#discussion_r2604494478